docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -4283,12 +4283,20 @@ Limitations
 
 Speaking of the names for employers, we will generate names for
 employers based on a conditional random model that Abie developed and
-Jim refactored into the vivarium model.  It should have more details
-documented at some point.  Jim will ensure that all employer names are
-unique for now, although it would be interesting one day to add some
-duplicates.  Businesses can have the same name in real life but
-if/when we do have duplicate names we need to prevent duplicates for
-the names of the largest employers.
+Jim refactored into the vivarium model. We based our simulated employer names 
```

Comment:
Not sure if it's out of scope for this PR, but we should move this -- I think the end of the "Initialization" section above would be the right place. We can also edit it to be more like the rest of our documentation (e.g. no need for a transition like "Speaking of..." and we don't need to say _who_ did this part).

There is also a TODO above about "Document how we name businesses" which should be removed.

Reply:
> and we don't need to say _who_ did this part

Is there any harm in saying who did something? Seems like that could potentially be useful information.

Reply:
I guess there's no harm in it; I think this document aspires to be something external facing and in order to fully get there it would have to explain who these people are instead of just using their first names. But we can leave the names as they are for now, since it's certainly not making anything worse.